indexing-service/src/main/java/org/apache/druid/indexing/common/task/batch/parallel/PartitionStat.java

```diff
@@ -30,6 +33,11 @@
  * set of data of the same time chunk (primary partition key) and the same secondary partition key
  * ({@link T}). This class holds the statistics of a single partition created by a task.
  */
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
```

Comment:
Should the default implementation be set to GenericPartitionStat for backward compatibility during rolling upgrade?